src/components/projects-summary/projects-summary.html

```diff
@@ -55,10 +49,13 @@ <h3 class="project-tile-header">
           <a href="" ng-click="$ctrl.handleProjectClick(project)" class="project-title">{{project | displayName}}</a>
         </h3>
         <p class="project-date">
+          <span ng-if="project | displayName : true"><span ng-bind-html="project.metadata.name | highlightKeywords : keywords"></span> &ndash;</span>
+          created
+          <span ng-if="project | annotation : 'openshift.io/requester'">by <span ng-bind-html="project | annotation : 'openshift.io/requester' | highlightKeywords : keywords"></span></span>
           {{project.metadata.creationTimestamp | date: 'mediumDate'}}
```

Comment:
This isn't new, but was it intentional to switch from relative timestamps? The project list page currently uses strings like "2 days ago"